From c86fcaeb22194699265fd26ff84c6fc1eb02d08d Mon Sep 17 00:00:00 2001
From: Paul Hunt <hunt@ti.com>
Date: Mon, 7 Mar 2011 17:10:37 -0700
Subject: syslink: ipu_pm: relax fixed latency constraint

Relax the latency constraint on Core (which is
asserted while ducati is running).

Change-Id: I20a64c7330e1850ecc9a226b0619d227052c6e8f
Signed-off-by: Paul Hunt <hunt@ti.com>
---
 drivers/dsp/syslink/ipu_pm/ipu_pm.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/dsp/syslink/ipu_pm/ipu_pm.h b/drivers/dsp/syslink/ipu_pm/ipu_pm.h
index 4ed30f4..1256865 100644
--- a/drivers/dsp/syslink/ipu_pm/ipu_pm.h
+++ b/drivers/dsp/syslink/ipu_pm/ipu_pm.h
@@ -110,7 +110,7 @@
 #define PM_CSTR_LAT_MASK	0x00000002
 #define PM_CSTR_BW_MASK		0x00000004
 
-#define IPU_PM_MM_MPU_LAT_CONSTRAINT	10
+#define IPU_PM_MM_MPU_LAT_CONSTRAINT	400
 #define IPU_PM_NO_MPU_LAT_CONSTRAINT	-1
 #define NO_FREQ_CONSTRAINT		0
 #define NO_LAT_CONSTRAINT		-1
-- 
1.6.0.4

